client-js/app/index.html

```diff
@@ -19,7 +19,7 @@
   <!-- Add to homescreen for Chrome on Android -->
   <meta name="mobile-web-app-capable" content="yes">
   <meta name="application-name" content="LabRAD">
-  <link rel="icon" sizes="192x192" href="/images/touch/chrome-touch-icon-192x192.png">
+  <link rel="icon" href="/images/labrad.png">
```

Comment:
That icon is actually from the [famfamfam silk icon set](http://www.famfamfam.com/lab/icons/silk/). I'd prefer to stick with polymer icons or stuff we make ourselves.